src/main/java/neo/UnMarkCommand.java

```diff
@@ -21,6 +21,10 @@ void complete(String tempi) throws NeoException, IOException {
         arrayLL.getTask(tempii-1).setIsDone(false);
         System.out.println("ohk I've marked this task as not done");
         System.out.println(arrayLL.getTask(tempii-1).toString());
+        stor.Writedata(arrayLL.getTask(0));
+        for(int i =1; i<arrayLL.arrayL.size(); i++) {
```

Comment:
Similar issue for this line of code, should be written as: `for (int i = 1; i < arrayLL.arrayL.size(); i++) {` based on java coding standard